proposals/subgroups.md

```diff
@@ -0,0 +1,227 @@
+# Subgroups
+
+Status: **Draft**
+
+Last modified: 2023-11-07
+
+Issue: #4306
+
+# Requirements
+
+**Vulkan**:
+* Vulkan 1.1, and
+* subgroupSupportedStages includes compute and fragment bit, and
+* subgroupSupportedOperations includes the following bits: basic, vote, ballot, shuffle, shuffle relative, arithmetic, quad, and
+* Vulkan 1.3 or VK_EXT_subgroup_size_control
```

Comment:
It would be worth mentioning that the subgroup functionality requires SPIR-V 1.3.